cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/IndexTest.scala

```diff
@@ -153,6 +153,22 @@ CREATE FULLTEXT INDEX rel_fulltext_index_name FOR ()-[r:HAS_PET|BROUGHT_PET]-()
 
 Create a fulltext index on relationships with the name `rel_fulltext_index_name`. Fulltext indexes on relationships can only be used by from the procedure `db.index.fulltext.queryRelationships`.
 
+###assertion=create-index
+//
+
+CREATE TEXT INDEX FOR (f:Friend) ON (f.email)
+###
+
+Create a text index on nodes with label `Friend` and property `email`.
+
+###assertion=create-index
+//
+
+CREATE TEXT INDEX text_index_name FOR ()-[h:HAS_PET]-() ON (h.since)
```

Comment:
Related to you comment about text index only being on strings, just realised that `since` might not be the best example here since that is something you would typically model with a temporal or a number. Maybe change it to the name of the pet or something?

Reply:
I tried to find something that fitted on a relationship but it did not go well :P